src/files/coverageparser.ts

```diff
@@ -200,11 +200,14 @@ export class CoverageParser {
         }
     }
 
-    private lcovExtract(lcovFile: string) {
+    private lcovExtract(filename: string, lcovFile: string) {
         return new Promise<Map<string, Section>>((resolve, reject) => {
             try {
                 source(lcovFile, async (err, data) => {
-                    if (err) { return reject(err); }
+                    if (err) {
+                        this.handleError("lcov-parse", err, filename);
```

Comment:
Similar to I mentioned below can we prepend to the error instead, so we dont leak information to the `handleError` function? I am hoping to refactor the handleError function here soon to be even more generic and reusable across the extension so not having specific info like filename will help greatly :smile: .
```
  error.message = `filename: ${filename} ${error.message}`
  this.handleError("lcov-parse", error);
```